### PR TITLE
feat: add beacon block proof test vectors (pre-Capella)

### PR DESCRIPTION
**Pull request by ogenev**
Generated 3 random beacon block proofs for pre-Cappella EL headers.

### CODE REVIEW

**Review by KolbyML — APPROVED** [review]

**Review by njgheorghita — APPROVED** [review]
Should we also add these as test vectors to the specs repo?

File: proofs/beacon_block_proofs_bellatrix/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Beacon Chain Block Proofs Bellatrix
+
+This folder contains beacon block proofs for post-Merge and pre-Cappella EL headers.
```
Comment (kdeme) [inline proofs/beacon_block_proofs_bellatrix/README.md:3]:
```suggestion
This folder contains beacon block proofs for post-Merge and pre-Capella EL headers.
```

File: proofs/beacon_block_proofs_bellatrix/beacon_block_proof-15539558-cdf9ed89b0c43cda17398dc4da9cfc505e5ccd19f7c39e3b43474180f1051e01.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# EL block header: 0xcdf9ed89b0c43cda17398dc4da9cfc505e5ccd19f7c39e3b43474180f1051e01
```
Comment (kdeme) [inline proofs/beacon_block_proofs_bellatrix/beacon_block_proof-15539558-cdf9ed89b0c43cda17398dc4da9cfc505e5ccd19f7c39e3b43474180f1051e01.yaml:1]: How are you getting the block hash in your tests? Do you get it from the filename?

Perhaps would be slightly better if it was also in the test vectors as either separate file or another object in the same file?

**Review by kdeme — COMMENTED** [review]
I'm currently having failures on the `beacon_block_header_proof` and `historical_roots_proof`. I might be doing something wrong, I'll further debug or perhaps try to generate them also.

File: proofs/beacon_block_proofs_bellatrix/beacon_block_proof-15539558-cdf9ed89b0c43cda17398dc4da9cfc505e5ccd19f7c39e3b43474180f1051e01.yaml (hunk 2 above)
Comment (ogenev) [reply proofs/beacon_block_proofs_bellatrix/beacon_block_proof-15539558-cdf9ed89b0c43cda17398dc4da9cfc505e5ccd19f7c39e3b43474180f1051e01.yaml:1]: I will add the block hash as another object in the YAML file.

**Comment by kdeme** [comment]
> I'm currently having failures on the `beacon_block_header_proof` and `historical_roots_proof`. I might be doing something wrong, I'll further debug or perhaps try to generate them also.

I had a silly error in the yaml parsing, the test vectors work for me :+1:. Would be nice to still get the block hash included in the files.


Another question: Why are the test vectors not put under the `tests/mainnet` folder? For example `tests/mainnet/history/block_proofs_bellatrix/` or similar?

**Comment by ogenev** [comment]
> Another question: Why are the test vectors not put under the `tests/mainnet` folder? For example `tests/mainnet/history/block_proofs_bellatrix/` or similar?

Fixed.

**Review by kdeme — APPROVED** [review]

**Pull request merged**